jenkinsapi/build.py

```diff
@@ -139,8 +139,12 @@ def get_changeset_items(self):
             "user": "username"
         }
         """
-        if 'items' in self._data['changeSet']:
-            return self._data['changeSet']['items']
+        if 'changeset' in self._data:
+            if 'items' in self._data['changeSet']:
+                return self._data['changeSet']['items']
+        elif 'changesets' in self._data:
```

Comment:
Need to double check this, but I think if you have multiple SCMs defined in your Jenkins job (be it Pipeline or regular one) you can end up having both `changeSet` and `changeSets` in the API.

In order to get a complete view of the changes included in the build both would need to be parsed and merged before returning.

Reply:
Updating before I forget.
Just tested regular and pipeline with 2 GIT scms.  
The regular returned changeSet with a list of items for both repos.
Pipeline job returned changeSets with a list of items for both repos.

Will spin up an svn server docker image and see what happens with an svn repo.

Reply:
I tried multi scm regular and pipeline with svn and git.
In both types of jobs it's just either changeSet or changeSets returned by the api.  Didn't see an occurrence where both were provided.